src/utils/accessibility/accessibility.js

```diff
@@ -3,20 +3,32 @@ function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? fontSize : Math.round(fontSize * 0.75);
+}
+
+function fontSizeInPx(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? Math.round(fontSize * 1.333) : fontSize;
+}
+
+
+function accessibleContrast(accessibilityLevel, fontSize, isFontBold, fontUnitOfMeasure) {
   switch (accessibilityLevel) {
     // http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast
     case 'AA':
-      return isLargeScale(fontSize, isFontBold) ? 3 : 4.5;
+      return isLargeScale(fontSizeInPt(fontSize,fontUnitOfMeasure), isFontBold) ? 3 : 4.5;
 
     // http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast7
     case 'AAA':
-      return isLargeScale(fontSize, isFontBold) ? 4.5 : 7;
+      return isLargeScale(fontSizeInPt(fontSize,fontUnitOfMeasure), isFontBold) ? 4.5 : 7;
   }
 
   return null;
 }
 
 export {
-  accessibleContrast
+  accessibleContrast,
+  fontSizeInPx
```

Comment:
Usually, when we export a function from a module, we want to test it.
Can we add tests for `fontSizeInPx`?